Header header

float64[] X_objects 
float64[] Y_objects
float64[] radians
float64[] radius
bool[] green_bouy
bool[] red_bouy
int64[] zone



